plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/mapping/DefaultIdentifierMapping.java

```diff
@@ -65,26 +72,38 @@ public String toRemoteColumnName(Connection connection, String columnName)
         return toRemoteIdentifier(connection, columnName);
     }
 
-    private String toRemoteIdentifier(Connection connection, String identifier)
+    public static class DatabaseMetaDataRemoteIdentifierSupplier
```

Comment:
I was being lazy when I added this class. Ideally this class should be a top-level class.